esp/bindings/http/platform/httpbinding.cpp

```diff
@@ -674,14 +674,11 @@ void EspHttpBinding::populateRequest(CHttpRequest *request)
         ISecResourceList* settinglist = m_setting_authmap->getResourceList("*");
         if(settinglist == NULL)
             return ;
-        if (getEspLogLevel()>=LogMax)
-        {
-            StringBuffer s;
-            DBGLOG("Set security settings: %s", settinglist->toString(s).str());
-        }
+        StringBuffer s;
```

Comment:
These kinds of examples are very unfortunate.  We will now be doing the work to build this string, and using up memory, for every request.  In the past it would very rarely be enabled and the work and memory would be avoided.

"settinglist->toString(s).str()" is a complete serialization of the list content.

Reply:
There aren't many like this.  If you replaced the original if with:
```
if (!queryLogMsgManager()->rejectsCategory(LegacyMsgCatMax))
```
I assume that would default to rejecting the category, and the user would have to enable that category to log this expensive content?